# HG changeset patch
# User Rik <rik@octave.org>
# Date 1440853185 25200
#      Sat Aug 29 05:59:45 2015 -0700
# Node ID 33b03b06442bd95cc66e121013f4858ee996cb7a
# Parent  2f94652de9ffbce18713b13ec09a033ccac9cae7
doc: Add a docstring for __current_scope__.

* symtab.cc (__current_scope__, __dump_symtab_info__): Add docstring for
__current_scope__.  Add seealso link to __current_scope__ from
__dump_symtab_info__.

diff --git a/libinterp/corefcn/symtab.cc b/libinterp/corefcn/symtab.cc
--- a/libinterp/corefcn/symtab.cc
+++ b/libinterp/corefcn/symtab.cc
@@ -1724,18 +1724,19 @@ determine whether functions defined in f
 ## Test input validation
 %!error (ignore_function_time_stamp ("all", "all"))
 %!error (ignore_function_time_stamp ("UNKNOWN_VALUE"))
 %!error (ignore_function_time_stamp (42))
 */
 
 DEFUN (__current_scope__, , ,
        "-*- texinfo -*-\n\
-@deftypefn {Built-in Function} {[@var{scope}, @var{context}]} __dump_symtab_info__ ()\n\
-Undocumented internal function.\n\
+@deftypefn {Built-in Function} {[@var{scope}, @var{context}]} __current_scope__ ()\n\
+Return the current scope and context as integers.\n\
+@seealso{__dump_symtab_info__}\n\
 @end deftypefn")
 {
   octave_value_list retval;
 
   retval(1) = symbol_table::current_context ();
   retval(0) = symbol_table::current_scope ();
 
   return retval;
@@ -1743,16 +1744,17 @@ Undocumented internal function.\n\
 
 DEFUN (__dump_symtab_info__, args, ,
        "-*- texinfo -*-\n\
 @deftypefn  {Built-in Function} {} __dump_symtab_info__ ()\n\
 @deftypefnx {Built-in Function} {} __dump_symtab_info__ (@var{scope})\n\
 @deftypefnx {Built-in Function} {} __dump_symtab_info__ (\"scopes\")\n\
 @deftypefnx {Built-in Function} {} __dump_symtab_info__ (\"functions\")\n\
 Undocumented internal function.\n\
+@seealso{__current_scope__}\n\
 @end deftypefn")
 {
   octave_value retval;
 
   int nargin = args.length ();
 
   if (nargin == 0)
     {
